### ISSUE TITLE
aws_instance documentation is incorrect

### ISSUE THREAD

**Issue by guidodobboletta** [terraform-providers/terraform-provider-aws#14766]
The documentation here: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/instance#block-devices says that the default `volume_type` is `standard`. 

I have tested this and the default volume type created is actually `gp2`. The code here suggests it too: https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/resource_aws_instance.go#L1374

**Comment by bflad** [comment]
Hi @guidodobboletta 👋  Thank you for reporting this and sorry you ran across that outdated documentation. I do believe some time ago EC2 switched the default to `gp2`. If you or anyone is interested in adjusting the documentation file, it can be found in this repository in the [`website/docs/r/instance.html.markdown` file](https://github.com/terraform-providers/terraform-provider-aws/blob/master/website/docs/r/instance.html.markdown).

**Comment by guidodobboletta** [comment]
Yup! It was changed here https://aws.amazon.com/about-aws/whats-new/2019/07/ebs-default-volume-type-updated-to-gp2/

I've created a PR here: https://github.com/terraform-providers/terraform-provider-aws/pull/14767 !
